content/productivity/upgrade-manager/plugin.js

```diff
@@ -0,0 +1,100 @@
+// plugins/UpgradeManager.ts
+
```

Comment:
Can we get some explainer text at the top here. Think if a newbie finds this plugin what they would want to ask. What is r s d, how it works globally, on a timer (1min?)

Reply:
Ok. Will do!